atomic_reactor/__init__.py

```diff
@@ -15,7 +15,7 @@
 import sys
 import time
 
-from atomic_reactor.version import __version__
+from atomic_reactor.version import __version__  # noqa
```

Comment:
In future, would be nice to specifically silence the particular diagnostic e.g. "noqa:E501" instead of silencing everything.